### DISCUSSION TITLE
How to modify global setting for lineHeight?

### DISCUSSION THREAD

**Discussion by linghuccc** [tailwindlabs/tailwindcss#12694] [Help]
I have modified tailwind.config.js to be
`theme: {
lineHeight: {
none: '2',
},`
, but after I refresh the page, there does not seem to be any difference. I then tried 
`theme: {
lineHeight: {
none: 'leading-8',
},`
but there is still no change at all.

Please advise. Thank you!

**Comment by wongjn** [comment]
`theme.lineHeight.none` corresponds to the `leading-none` class. To have this apply to the page, you'd need to use the `leading-none` class somewhere.

Otherwise, if you wanted to change the default `line-height` that [Preflight](https://tailwindcss.com/docs/preflight) sets, you could:
- Apply a `leading-*` class to some high-level element of the page, like the `<html>` or `<body>`:
  ```html
  <html class="leading-loose">
  ```
- Apply `line-height` in your CSS:
  ```css
  @tailwind base;
  @tailwind components;
  @tailwind utilities;

  @layer base {
    html {
      line-height: 2;
    }
  }
  ```
- Use a [Tailwind Plugin](https://tailwindcss.com/docs/plugins#adding-base-styles):
  ```js
  const plugin = require('tailwindcss/plugin')

  module.exports = {
    // …
    plugins: [
      plugin(function({ addBase, theme }) {
        addBase({
          'html': {
            lineHeight: '2',
          },
        })
      })
    ]
  }
  ```
